### ISSUE TITLE
Alliance wins BG by losing (Anathema, horde)

### ISSUE THREAD

**Issue by jieuh** [LightsHope/issues#139]
Hey

We were in a wsg, Acelius, Zmile and Panting.
In front there were 3 other players, Torygg (warrior), a rogue and a mage.

I capped 1 flag, they capped 0.
Horde had 7 kills, when alliance had 0.
Yet at the end of the 5 minutes (because there were not enough players in BG), it says "The Alliance Wins"? We only got one mark, they had 3.

It is not the first time it ever happens, while having draw games (0-0) alliance keeps getting more honor. Any idea on whats up? Can we get our marks?
Thank you! :)

**Comment by GuybrushGit** [comment]
When the BG ends due to lack of players and one side has the minimum number of players that side wins.

**Comment by jieuh** [comment]
We just finished one, we were 3 (Stopper, Disastermaze, Panting) they were 2 (Loveangel and Xit).

We didn't win, "draw game". Even tho we marked one flag before the 5 minutes end.

**Comment by Niceguyone** [comment]
This guide (https://www.gamesradar.com/cheats/15196/ , last updated: 20 May 2006) has a quite accurate description of what should happen (at least in Warsong Gulch) if not enough players are in the game:

> Sometimes you're just doomed from the start, it seems. The first game of
> the day on the Scarlet Crusade server, for example, is almost always a 4 Horde
> versus 8 Alliance game. (A couple times I've even had to fight all of them
> alone!) What to do under these conditions? A few rules specific to unbalanced
> games: If, after the two minute period before the battle starts, the number of
> players is less than five on one side or the other, the game will begin a
> five-minute timer to end the battle. Every minute you'll see it counting down
> in yellow text (like a server shutdown) until the battle ends. When the battle
> does end, the winner is determined by who has the most capped flags. If neither
> side has capped a flag, the side that's carrying a flag wins. If both sides are
> carrying a flag, or if no one has one, the game is a draw. A draw game is the
> worst possible outcome of a Gulch run. It's worse than losing, in terms of game
> logistics. You gain no honor. You don't get a mark. And to add insult to
> injury, you get a deserter flag. So sometimes it's better to surrender one of
> these broken little games at the end, if you're after the honor and feeling
> generous to your opponent.

In other words, the side with more points wins, otherwise it will be a draw, meaning that both sides get a Deserter debuff and gain no honor or marks.

**Comment by jieuh** [comment]
Yeah at the moment we get one mark everytime we lose, (timer or not). But as i noted on my OP alliance lost and had 3 marks :( :(

**Comment by jieuh** [comment]
Very good documentation btw
